ledger/controllers/surveyor.js

```diff
@@ -4,6 +4,9 @@ const boom = require('boom')
 const bson = require('bson')
 const underscore = require('underscore')
 const surveyors = require('../lib/surveyor')
+const {
+  BigNumber
+} = require('bat-utils/lib/extras-utils')
```

Comment:
Do you know why extras-utils has BigNumber as a export?  It seems like everywhere else in the project BigNumber is required like this

```
const BigNumber = require('bignumber.js')
```

Reply:
It looks like the reason is so we can explicitly set the number of decimal places and when we want display exponential notation.  If this is the preferred way, should we change the other places where BigNumber is required?

Reply:
yeah, so i had an issue for this to create a clone of bignumber with a higher version number that supports scoped constructors. removed for now